kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -10,9 +10,8 @@
         id="chan-select"
         name="chan-select"
         v-model="getCurrentChannel"
```

Comment:
I did the opposite, and made this work with `v-model` instead by having a getter and setter on the `getCurrentChannel` computed property. Given that this is an input field, having the two way data binding is helpful.